packages/shell-api/src/database.ts

```diff
@@ -1445,4 +1445,18 @@ export default class Database extends ShellApiWithMongoClass {
     this._mongo._instanceState.currentCursor = cursor;
     return cursor;
   }
+
+  @returnsPromise
+  @returnType('AggregationCursor')
+  async sql(sqlString: string, options?: Document): Promise<AggregationCursor> {
```

Comment:
I think you’ll need an entry for this in the i18n package as well (but CI will tell you for sure :))